data-reconciliation-app/test/unit-test/auth/validator/jwt/jwt-validation.test.ts

```diff
@@ -0,0 +1,51 @@
+// Use the CCF polyfill to mock-up all key-value map functionality for unit-test
+import * as ccfapp from "@microsoft/ccf-app";
+import exp from "constants";
+import jwtValidator from "../../../../../src/auth/validator/jwt/jwt-validation";
+import { MSAccessToken } from "../../../../../src/auth/validator/jwt/ms-aad-jwt-provider";
+import { MS_AAD_CONFIG } from "../../../../../src/utils/config";
+
+describe("Jwt-Validator", () => {
+  let iss = "https://login.microsoftonline.com/common/v2.0";
+  let aud = MS_AAD_CONFIG.ApiIdentifierUri;
+  let appid = MS_AAD_CONFIG.ClientApplicationId;
+  let ver = "1.0";
+
+  let header = { key1: "value1" };
+  let param = { param1: "value1" };
+  let test_body: ccfapp.Body = null;
```

Comment:
If these variables will not be changed in later stage, we could define it as `const`